Zilon.Core/Zilon.Core/Persons/HumanPerson.cs

```diff
@@ -21,120 +22,106 @@ namespace Zilon.Core.Persons
     /// <summary>
     /// Персонаж, находящийся под управлением игрока.
     /// </summary>
-    public class HumanPerson : IPerson
+    public class HumanPerson : PersonBase
     {
         private readonly ITacticalActScheme _defaultActScheme;
         private readonly ISurvivalRandomSource _survivalRandomSource;
 
         /// <inheritdoc/>
-        public int Id { get; set; }
+        public override int Id { get; set; }
 
         /// <inheritdoc/>
         public string Name { get; }
 
-        /// <inheritdoc/>
-        public IEquipmentCarrier EquipmentCarrier { get; }
-
-        /// <inheritdoc/>
-        public ITacticalActCarrier TacticalActCarrier { get; }
-
-        /// <inheritdoc/>
-        public IEvolutionData EvolutionData { get; }
-
         /// <inheritdoc/>
         public IPersonScheme Scheme { get; }
 
-        /// <inheritdoc/>
-        public ICombatStats CombatStats { get; }
-
-        /// <inheritdoc/>
-        public IPropStore Inventory { get; }
-
-        /// <inheritdoc/>
-        public ISurvivalData Survival { get; }
-
-        /// <inheritdoc/>
-        public EffectCollection Effects { get; }
-
         public IPlayerEventLogService PlayerEventLogService { get; set; }
 
-        public PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
-
-        public bool HasInventory { get => true; }
-
-        public IDiseaseData DiseaseData { get; }
+        public override PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5453268)